firefox_ui_tests/update/manifest.ini

```diff
@@ -0,0 +1,5 @@
+[test_direct_update.py]
+direct = true
```

Comment:
We can do this. The benefit would be that the other test for the other type is not marked as skipped. So I will test it, and if it works as expected I will move the tests to individual sub folders and create appropriate manifest files.